src/main/java/frc/lib/Constants.java

```diff
@@ -0,0 +1,46 @@
+package frc.lib;
+
+import frc.lib.doubleNeo.doubleNeoConstants;
+import frc.lib.krakentalon.krakenTalonConstants;
+
+public final class Constants extends krakenTalonConstants {
+
+
+    public static final class Intake {
+        public static final int intakeMotor = 32;
+        public static final double intakeSpeed = 0.8;
+    }
+
+    public static final class Arm {
+        public static final int armMotor = 33;
+        public static final int armFollowerMotor = 34;
+
+        public static final double kP = 1.0;
+        public static final double kI = 0.0;
+        public static final double kD = 0.0;
+        public static final double kF = 0.0;
+
+        public static final double desiredShooterAngle = 30.0;
+        public static final double desiredIntakeAngle = -30;
+        public static final double armEncoderOffset = 0.0;
+
+        public static final double minAngle = 31.0;
+        public static final double maxAngle = 31.0;
+    }
```

Comment:
Instead of strict constants, try this

```suggestion
    public class ArmConfig {
        public final int armMotor;
        public final int armFollowerMotor;

        public final double kP;
        public final double kI;
        public final double kD;
        public final double kF;

        public final double desiredShooterAngle;
        public final double desiredIntakeAngle;
        public final double armEncoderOffset;

        public final double minAngle;
        public final double maxAngle;
        
        // Default constructor uses all default values
        public ArmConfig() {
            this(33, 34);
        }
        
        // 
        public ArmConfig(int armMotorId, int armFollowerMotorId) {
            this(
                armMotorId,
                armFollowerMotorId,
                1.0,
                0.0,
                0.0,
                0.0
            );
        }
        
        public ArmConfig(
            int armMotorId,
            int armFollowerMotorId,
            double kP,
            double kI,
            double kD,
            double kF
        ) {
            this(
                armMotorId,
                armFollowerMotorId,
                kP,
                kI,
                kD,
                kF,
                30.0, // desiredShooterAngle default
                -30.0, // desiredIntakeAngle default
                0.0 // armEncoderOffset default
            );
        }
        
        public ArmConfig(
            int armMotorId,
            int armFollowerMotorId,
            double kP,
            double kI,
            double kD,
            double kF,
            double desiredShooterAngle,
            double desiredIntakeAngle,
            double armEncoderOffset
        ) {
            this.armMotor = armMotorId;
            this.armFollowerMotor = armFollowerMotorId;
            this.kP = kP;
            this.kI = kI;
            this.kD = kD;
            this.kF = kF;
            this.desiredShooterAngle = desiredShooterAngle;
            this.desiredIntakeAngle = desiredIntakeAngle;
            this.armEncoderOffset = armEncoderOffset;
        }
    }
```